docs/contributing/how-to-contribute.md

```diff
@@ -101,7 +101,7 @@ In order to accept your pull request, we need you to submit a CLA. You only need
 
 ### Development Workflow
 
-After cloning React, run `npm install` to fetch its dependencies.  
```

Comment:
Doesn't matter much here, but for the future reference: whitespace is significant in Markdown. So if you see two spaces at the line end, it is likely intentional to break the line.